web/packages/teleport/src/services/auth/auth.ts

```diff
@@ -315,15 +298,55 @@ const auth = {
     return api.post(cfg.api.createPrivilegeTokenPath, {});
   },
 
-  getWebauthnResponse(
+  async getWebauthnResponse(
     scope: MFAChallengeScope,
     allowReuse?: boolean,
     isMFARequiredRequest?: IsMfaRequiredRequest
   ) {
+    // TODO(Joerger): DELETE IN 16.0.0
+    // the create mfa challenge endpoint below supports
+    // MFARequired requests without the extra roundtrip.
+    if (isMFARequiredRequest) {
+      try {
+        const isMFARequired = await checkMfaRequired(isMFARequiredRequest);
+        if (!isMFARequired.required) {
+          return;
+        }
+      } catch (err) {
+        if (
+          err?.response?.status === 400 &&
+          err?.message.includes('missing target for MFA check')
```

Comment:
I would put a comment on the Go side where this error is defined that says that the UI depends on this not changing.

Reply:
The UI only depends on this for old versions, new versions will know how to handle the `IsMfaRequiredAdminAction` request.